molecule/default/molecule.yml

```
@@ -0,0 +1,53 @@
+---
+dependency:
+  name: galaxy
+  enabled: true
+driver:
+  name: podman
+platforms:
+  - name: opencast-activemq-centos8
+    image: quay.io/centos/centos:stream8
+    pre_build_image: false
+    command: /sbin/init
+    tmpfs:
+      - /run
+      - /tmp
```

`containers` driver expects a dict here

```suggestion
      "/tmp": "exec"
      "/run": "rw,noexec,nosuid,nodev"
```